server/main.lua

```diff
@@ -69,31 +69,41 @@ exports('UpdateVehicleEntity', updateEntity)
 
 ---@class SetEntityOwnerQuery
 ---@field citizenId string
+---@field license string
 ---@field vehiclePlate string
 
 --- Update Vehicle Entity Owner
 ---@param query SetEntityOwnerQuery
 local function setEntityOwner(query)
-    MySQL.update('UPDATE player_vehicles SET citizenid = ? WHERE plate = ?', {
+    MySQL.update('UPDATE player_vehicles SET citizenid = ?, license = ? WHERE plate = ?', {
         query.citizenId,
+        query.license,
         query.vehiclePlate
```

Comment:
```suggestion
    MySQL.update('UPDATE player_vehicles INNER JOIN players ON players.citizenid = @citizenid SET player_vehicles.citizenid = @citizenid player_vehicles.license = players.license WHERE player_vehicles.plate = @plate', {
    citizenid = query.citizenid,
    plate = query.vehiclePlate,
```

This is what I meant. I think the syntax is correct. Will need to be tested to be sure.